### DISCUSSION TITLE
Course outside of fields (border...)

### DISCUSSION THREAD

**Discussion by elkooo** [Courseplay/Courseplay_FS22#3017] [Ideas]
Hello, 

I just started with Courseplay and played around with its functions. 
While I use the fields to do crop, I oftentimes also mow around borders (because you also own the boarders and you can get a substancial amount of gras or hey out of the border section of some pieces of propperty and not just the crop of the field). 

So I tried the record a course (which worked) and saved it (it was mentioned that it will be saved as CP-1). Later I tried to load that course again but this didn't work - I think what it made was just a «border» for a field - not an actual course... so would it be possible to record a course and let a mower just follow the course even though it is not on an actual field?

you see the cp1 course just above and left of field 37... I was able to record it but not to recall it...

![cp1](https://github.com/Courseplay/Courseplay_FS22/assets/6999860/c2d58a14-50a9-4d0e-9cac-6edf72bb1005)

**Comment by Tensuko** [comment]
Recording a course is not part of CP for FS22.
For outside the field, use AutoDrive.

**Reply by elkooo** [reply]
ok thanks for your reply.
would be nice though :) the recording feature also would be neat if you prefer to do a course on a field manually...

thanks anyway for all the work put in to courseplay! it's fun to try it... mostly works great (except one time, when it drove my tractor straight into the river instead of having it unload the harvester...)
